pkg/runtime/wfengine/backends/actors/backend_test.go

```diff
@@ -208,6 +209,18 @@ func TestResetLoadedState(t *testing.T) {
 	assert.Equal(t, 15, deleteCount) // all history and inbox records are deleted
 }
 
+func TestInvalidStart(t *testing.T) {
+	be, err := NewActorBackend(wfbackend.Metadata{}, nil)
+	require.NoError(t, err)
+	require.NotNil(t, be)
+
+	err = be.Start(context.TODO())
+	require.NoError(t, err)
+
+	err = be.Start(context.TODO())
```

Comment:
Should these not be errors if we are giving an invalid start?

Reply:
🤦